inappmessaging/USERGUIDE.md

```diff
@@ -323,6 +322,10 @@ Documents targeting Product Managers:
 
 ## <a name="changelog"></a> Changelog
 
+### 4.1.1-PP (in-progress)
+* SDKCF-3781: Fixed crash due to failing master key generation for encrypted shared preferences.
+  - The SDK will try to create a master key again after the first failed attempt, and will fallback to normal shared preferences if the key generation still failed.
```

Comment:
'still failed' or 'still fails'? (I'm not sure :D )

Reply:
updated to "still fails"